packages/sparkline/package.json

```diff
@@ -32,20 +41,15 @@
   },
   "homepage": "https://github.com/williaster/data-ui#readme",
   "devDependencies": {
-    "babel-jest": "^20.0.3",
-    "babel-polyfill": "^6.23.0",
-    "enzyme": "^3.1.0",
-    "enzyme-adapter-react-16": "^1.0.0",
-    "jest": "^20.0.3",
+    "@beemo/cli": "^0.19.0",
```

Comment:
Installed these as `@data-ui/build-config` alone wasn't sufficient for running `beemo` scripts, could push to the root `package.json`

Reply:
Does `lerna bootstrap` not copy the `beemo` bin to the root?